docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -124,6 +124,27 @@ az login --identity --username <client_id|object_id|resource_id>
 
 To learn more about managed identities for Azure resources, see [Configure managed identities for Azure resources](/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm) and [Use managed identities for Azure resources for sign in](/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-sign-in).
 
+## Sign in with WAM
+
+The Azure CLI now offers preview support for Web Account Manager (WAM).  WAM is a Windows 10+ component that acts as an authentication broker.  Using WAM has several benefits:
+
+_ Enhanced security. See [Conditional Access: Token protection (preview)](/azure/active-directory/conditional-access/concept-token-protection).
+_ Support for Windows Hello, conditional access, and FIDO keys.
+_ Fast single sign-on.
```

Comment:
"Fast" seems too subjective. I'd make this more about the platform SSO benefits. 

```suggestion
_ Streamlined single sign-on.
```